src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -27,17 +29,21 @@ public class HttpProtocolTests : HttpClientTest
         [InlineData("HTTP/1.1 500 Internal Server Error", 500, "Internal Server Error")]
         [InlineData("HTTP/1.1 555 we just don't like you", 555, "we just don't like you")]
         [InlineData("HTTP/1.1 600 still valid", 600, "still valid")]
-        // TODO #24713: The following pass on Windows on .NET Core but fail on .NET Framework.
```

Comment:
Why remove these test cases if the test is skipped on netframework?

Reply:
Because now the test cases will fail on .NET Core and ManagedHandler, the behavior is consistent with .NET Framework.